NEWS.md

```diff
@@ -1,6 +1,10 @@
-# Revisions and Change Log
+## 0.1.3
```

Comment:
You need a line for 0.1.4, you already have 0.1.3 released, so there should be an 0.1.3 and an 0.1.4 line, every release needs a news line.

Reply:
done